{"uuid": "1a1a3494-48ca-4fd6-86b8-fb3b2ccac1e2", "children": ["c99eecd7-6e4b-4428-b313-0aac3d402af9", "b45b58d2-1e42-4891-a868-de0a7cc723b4", "75db3182-00e9-4a04-b53a-71321e5a18da"], "befores": [{"name": "initialize_test", "status": "passed", "start": 1699537004264, "stop": 1699537004264}], "afters": [{"name": "initialize_test::0", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=119.0.6045.123)\nStacktrace:\n0   chromedriver                        0x00000001032d3d28 chromedriver + 4795688\n1   chromedriver                        0x00000001032cb2b3 chromedriver + 4760243\n2   chromedriver                        0x0000000102ea488d chromedriver + 407693\n3   chromedriver                        0x0000000102e78758 chromedriver + 227160\n4   chromedriver                        0x0000000102f2050f chromedriver + 914703\n5   chromedriver                        0x0000000102f26da3 chromedriver + 941475\n6   chromedriver                        0x0000000102f1ac68 chromedriver + 892008\n7   chromedriver                        0x0000000102ee5143 chromedriver + 672067\n8   chromedriver                        0x0000000102ee631e chromedriver + 676638\n9   chromedriver                        0x0000000103294795 chromedriver + 4536213\n10  chromedriver                        0x0000000103299853 chromedriver + 4556883\n11  chromedriver                        0x000000010327a001 chromedriver + 4427777\n12  chromedriver                        0x000000010329a59d chromedriver + 4560285\n13  chromedriver                        0x000000010326b48c chromedriver + 4367500\n14  chromedriver                        0x00000001032ba0e8 chromedriver + 4690152\n15  chromedriver                        0x00000001032ba29e chromedriver + 4690590\n16  chromedriver                        0x00000001032caeee chromedriver + 4759278\n17  libsystem_pthread.dylib             0x00007ff80bc25202 _pthread_start + 99\n18  libsystem_pthread.dylib             0x00007ff80bc20bab thread_start + 15\n\n", "trace": "  File \"/Users/DerekPetersen/PycharmProjects/CIG/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/DerekPetersen/PycharmProjects/CIG/venv/lib/python3.9/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/DerekPetersen/PycharmProjects/CIG/conftest.py\", line 10, in initialize_test\n    base_page.get_driver().close()\n  File \"/Users/DerekPetersen/PycharmProjects/CIG/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 455, in close\n    self.execute(Command.CLOSE)\n  File \"/Users/DerekPetersen/PycharmProjects/CIG/venv/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/DerekPetersen/PycharmProjects/CIG/venv/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1699537009364, "stop": 1699537009370}], "start": 1699537004264, "stop": 1699537009371}